src/components/CreateNewVM.vue

```diff
@@ -79,6 +79,10 @@
             <q-input v-model="newProgram.entrypoint" label="Your entrypoint"
             stack-label standout class="q-my-sm" />
         </div>
+        <div class="col-12">
+            <q-input v-model="newProgram.metadata.name" label="Name your VM"
```

Comment:
The term VM refers to how the program is running technically, but it could change to become containers/... .

A user uploads a program on Aleph, and the program is executed inside a VM.


```suggestion
            <q-input v-model="newProgram.metadata.name" label="Name your Program"
```